src/pages/index.astro

```diff
@@ -15,7 +15,7 @@ import "../styles/base.css";
         <div class="w-full mx-auto max-w-screen-md space-y-8 px-6 md:px-8">
           <h1 class="text-6xl font-bold tracking-tight">Tech for Palestine</h1>
           <p class="text-lg md:text-xl">
-            A loose coalition of over 40 founders, engineers, product marketers,
+            A loose coalition of over 2000+ founders, engineers, product marketers,
```

Comment:
It should be either `over 2000` or `2000+` what's your thought? @pbiggar

Reply:
2000+ I think? Both are fine, pick your fav